.jenkins/pipelines/web.groovy

```diff
@@ -0,0 +1,80 @@
+pipeline {
+    agent {
+        docker {
+            image 'testmnrp/flutter-chromedriver:3'
+            args '-u root:root'
+        }
+    }
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+    environment {
+        METRICS_PROJECT_ID = "jenkins_metrics_web"
+    }
+    stages {
+        stage('Git clone') {
+            steps {
+                git branch: 'master', url: 'https://github.com/platform-platform/monorepo.git'
+            }
+        }
+        stage('Get Dependencies') {
+            steps {
+                dir('metrics/web'){
+                    sh 'flutter config --enable-web'
+                    sh 'pub get'
+                }
+                script{
+                    isTested = false
+                }
+            }
+        }
+        stage('Parallel Stage') {
+                    parallel {
+                        stage('Run web driver tests') {
+                            steps {
+                                dir('metrics/web'){
+                                    withCredentials([usernamePassword(credentialsId: 'app_credentials', passwordVariable: 'PASSWORD', usernameVariable: 'EMAIL')]) {
+                                        sh 'dart test_driver/main.dart --verbose --store-logs-to=build/logs --email=$EMAIL --password=$PASSWORD --no-verbose'
+                                    }
+                                }
+                            }
+                        }
+                        stage('Run tests') {
+                            steps {
+                                dir('metrics/web'){
+                                    sh 'flutter test -v --coverage --coverage-path build/coverage.info'
+                                }
+                                script{
+                                    isTested = true
+                                }
+                            }
+                        }
+
+                    }
+                }
+        stage('Convert coverage report') {
+            steps {
+                waitUntil{
+                    script{
+                        return isTested
+                    }
+                }
```

Comment:
Can we remove `waitUntil` here since this stage goes after `Parallel Stage` and will be run only when it finishes?